### ISSUE TITLE
Watched custom tokens not updating amount, not showing in wallet, can't send - but they show in recent transactions.

### ISSUE THREAD

**Issue by hiimphil** [ethereum/mist#2939]
``````
Version: `0.9.0`
OS & Version: osx 10.12.6
Node type: `geth`
Number of blocks synchronized: 4,155,735
``````

Hey there. My Mist wallet is synced, I have transferred ether in and out of it, but I'm having trouble with custom tokens. I have added the watch tokens information and they come up not greyed but the amounts don't update...

<img width="1130" alt="screen shot 2017-08-13 at 11 46 12 pm" src="https://user-images.githubusercontent.com/30997708/29260936-b5586d44-8081-11e7-9db8-f94e7c269a66.png">

once I added the tokens under custom token, the "recent transactions" came through and showed both custom token deposits

<img width="1105" alt="screen shot 2017-08-13 at 11 48 25 pm" src="https://user-images.githubusercontent.com/30997708/29261041-514d3b62-8082-11e7-87cf-251df998d986.png">

but the tokens don't show in my wallet and they aren't listed anywhere when I go to Send. Any ideas?

<img width="1189" alt="screen shot 2017-08-13 at 11 52 48 pm" src="https://user-images.githubusercontent.com/30997708/29261136-cdc1a16a-8082-11e7-92c2-b1bb95df675f.png">

thanks in advance.

**Comment by sergejs99** [comment]
hi have u got that sorted?i have same problem with EOS

**Comment by hiimphil** [comment]
No. I was able to log into my wallet via MEW online however and can send my
tokens from there.

*Phillip Van Wagoner*
C: (517)862-5858 <(517)%20862-5858>
